exporter/dynatraceexporter/README.md

```diff
@@ -1,24 +1,27 @@
-[Dynatrace](https://www.dynatrace.com/integrations/opentelemetry/) can receive OpenTelemetry metrics in the following ways:
-
-- Using OTLP/HTTP (OpenTelemetry Protocol) metrics exporter (recommended)
-- Using Dynatrace OpenTelemetry metrics exporter described below
-
-More information on exporting metrics to Dynatrace can be found in the
-[Dynatrace documentation for OpenTelemetry metrics](https://www.dynatrace.com/support/help/shortlink/opentelemetry-metrics).
-
-For ingesting traces (spans) into Dynatrace, use the generic OTLP/HTTP exporter shipped with the Collector.  
-More information on exporting traces to Dynatrace can be found in the
-[Dynatrace documentation for OpenTelemetry traces](https://www.dynatrace.com/support/help/extend-dynatrace/opentelemetry/opentelemetry-traces/opentelemetry-ingest).
+[Dynatrace](https://www.dynatrace.com/integrations/opentelemetry) supports native
+OpenTelemetry protocol (OTLP) ingest for traces, metrics and logs.
+All signals can be sent directly to Dynatrace via **OTLP protobuf over HTTP**
+using the built-in OTLP/HTTP Exporter available in the collector.
+More information on using the collector with Dynatrace can be found in the
+[Dynatrace documentation](https://www.dynatrace.com/support/help/shortlink/opentelemetry-collector-explanation#configuration).
 
 # Dynatrace Exporter
```

Comment:
The first paragraph might be overlooked too easily and the extra text above the first heading looks a bit odd I think, also most markdown linters would even complain about that.

How about adding this?


```suggestion
# Dynatrace

[Dynatrace](https://www.dynatrace.com/integrations/opentelemetry) supports native
OpenTelemetry protocol (OTLP) ingest for traces, metrics and logs.
All signals can be sent directly to Dynatrace via **OTLP protobuf over HTTP**
using the built-in OTLP/HTTP Exporter available in the collector.
More information on using the collector with Dynatrace can be found in the
[Dynatrace documentation](https://www.dynatrace.com/support/help/shortlink/opentelemetry-collector-explanation#configuration).

## Dynatrace Metrics Exporter
```
(+ adding an extra # to the others headings below)